Makefile

```diff
@@ -24,6 +24,7 @@ SUBDIRS:=ctriface taps misc
 EXTRAGOARGS:=-v -race -cover
 EXTRAGOARGS_NORACE:=-v
 EXTRATESTFILES:=fccd-orchestrator_test.go stats.go fccd-orchestrator.go functions.go
+SKIPTestAllFunctions:=-short
```

Comment:
let us just specify -short in case we will be excluding more tests later

Reply:
done